### DISCUSSION TITLE
Issues with webhooks

### DISCUSSION THREAD

**Discussion by burnoutberni** [Budibase/budibase#1894] [Help] [answered]
I already had an issues opened (#1854), but I thought I had figured out to the things that I want, namely:
I have a table called "Postings". Whenever a row gets added, the whole content of the new row should be pushed as JSON to another API. After feeling lost for a while, I figured out that I can use `{{JSONstringify trigger.row}}` to do exactly what I want – in theory.

In practice, this only works when testing the automation, but not with the actual table and I don't really know why. I'm starting to feel like this is a bug, but I don't wanna reopen my original bug report, because it's to convoluted. Also, maybe once again I'm doing something wrong, so I'd like to ask for your help first.

This is what my automation looks like (with the above JSON body):
![Screenshot from 2021-07-01 12-43-39](https://user-images.githubusercontent.com/1640933/124111850-029a7e80-da6a-11eb-8703-da713a347868.png)

When I test the automation, I receive the following JSOn:
```
{
  'Auto ID': 1,
  'Created At': 'TEST',
  'Updated At': 'TEST',
  Titel: 'TEST',
  Text: 'TEST',
  'Eingeplant für': 'TEST'
}
```

When I actually add a row, nothing happens. Is there any sort of log or something which could be used for debugging the webhook?

Thanks in advance for your help!

**Comment by mjashanks** [comment, marked as answer]
Hey @burnoutberni 

I just tested this exactly as you detailed above, using a Budibase internal table, and it worked for me. Are you using a Budibase table?

Is it possible that you have not enabled your automation (below)? - this trips me up sometimes!

![image](https://user-images.githubusercontent.com/3524181/124131345-9cb4f380-da77-11eb-8b8a-b0799810a6b3.png)

**Reply by burnoutberni** [reply]
Oh shoot, that does feel rather awkward – but yes, that seems to be what was missing. Interestingly I must have activated the automation before by mistake, because it did work at some point, and I wasn't aware at all that it needs to be activated. Anyways, thanks a lot! :)

**Reply by mjashanks** [reply]
No problem - It's a UX issue that we need to fix. Everyone does it!
